### DISCUSSION TITLE
Building latest version fails because of missing dependency "imagefilters"

### DISCUSSION THREAD

**Discussion by centic9** [FossifyOrg/General-Discussion#389] [Q&A] [answered]
When I try to build the current master I get an error about a missing dependency:

```
Execution failed for task ':app:dataBindingMergeDependencyArtifactsFossDebug'.
> Could not resolve all files for configuration ':app:fossDebugCompileClasspath'.
   > Could not find info.androidhive:imagefilters:1.0.7.
     Required by:
         project :app

```

The maven repository lists it, but not as part of MavenCentral: https://mvnrepository.com/artifact/info.androidhive/imagefilters/1.0.7

Is there anything else needed to build this project?

**Comment by naveensingh** [comment]
Have you made any changes to the build scripts?

**Reply by centic9** [reply]
No, I just cloned the repo and then called `./gradlew buildDebug`.

Opening the project in Android Studio shows the same error.

**Comment by nekotrix** [comment]
Same error on my end.

**Comment by naveensingh** [comment]
Couldn't reproduce this. I'm currently working on the Gallery but haven't faced this issue even one time. Fossify will likely fork this and other abandoned libraries the project depends upon. In the meantime, you guys can try forking the project and building it from jitpack.io: https://github.com/emonm/AndroidPhotoFilters

**Reply by obfusk** [reply]
You probably have it cached locally; dependencies like `info.androidhive:imagefilters:1.0.7` that used `repo.spring.io` are [no longer available](https://spring.io/blog/2022/12/14/notice-of-permissions-changes-to-repo-spring-io-january-2023/).

IzzyOnDroid currently cannot confirm a Reproducible Build for `1.2.0` (and e.g. F-Droid will not be able to build at all).

cc @IzzySoft

**Comment by naveensingh** [comment]
Ok, I'll fix this soon and release a new update.

**Reply by naveensingh** [reply, marked as answer]
@centic9 @trixoniisama @obfusk 

It's fixed now, thanks!

**Reply by IzzySoft** [reply]
Late reply (I was on vacation): RB is in place again, too (did that remotely from Cyprus on Sept 28th, but could not respond here earlier) :smiley:

**Comment by centic9** [comment]
Thanks, building locally works now!
